### PR TITLE
Update runtime, libraries and application + add aarch64

### PR DESCRIPTION
**Pull request by lionirdeadman**
This needs testing

### CODE REVIEW

**Comment by hfiguiere** [comment]
This also build on aarch64.

(the patch isn't upstreamable, unless they decide that on linux the non x86_64 is aarch64, which IMHO make more sense than i386)

**Comment by hfiguiere** [comment]
not sure about the failure, let me check

**Comment by hfiguiere** [comment]
ah yeah my patch was broken.

File: net.mediaarea.QCTools.json [hunk 1]
```diff
@@ -58,19 +59,16 @@
...
-          "url": "https://mediaarea.net/download/source/qctools/1.1/qctools_1.1.tar.xz",
-          "sha256": "0114312e64dd489c4f68f2b78a27a05163e64a98644d2e9d5c50b9c154905df7"
-        },
-        {
-            "type": "patch",
-            "path": "use-system-libraries.patch",
-            "options": [ "--binary" ]
+          "url": "https://mediaarea.net/download/source/qctools/1.2/qctools_1.2.tar.xz",
+          "sha256": "cc73f46a24c6181912c233be6508480e8f0e4b618052a447a864afd3e8d5a167"
         },
```
Comment (hfiguiere) [inline net.mediaarea.QCTools.json:72]:
```suggestion
        },
        {
            "type": "patch",
            "path": "aarch64-build.patch"
        },
```

This was missing.

**Review by g-maxime — APPROVED** [review]

**Comment by g-maxime** [comment]
@ lionirdeadman, @hfiguiere  thanks

**Pull request merged**